packages/palette/src/elements/LabeledRange/LabeledRange.tsx

```diff
@@ -57,22 +58,29 @@ export class LabeledRange extends React.Component<
   }
 
   render() {
-    const { formatter, label } = this.props
+    const { formatter, label, disabled, disabledText } = this.props
     const { min, max } = this.state
-
+    const useDisabledText = disabled && disabledText
     return (
       <Flex width="100%" flexDirection="column">
         <Header mt="-6px">
-          <Flex justifyContent="space-between">
-            <Sans size="2" color="black100" mt={0.3}>
-              {label}
-            </Sans>
-            <Sans size="2" color="black60" mt={0.3}>
-              {formatter
-                ? formatter(min, max, this.maxIndicator())
-                : this.toString()}
+          {!useDisabledText && (
```

Comment:
I think this makes more sense as a ternary, instead of two separate logical AND's. The two branches are currently mutually exclusive, and a ternary would make that more clear. Something like: 

```
<Header ... >
  {useDisabledText ? (
    <Sans>...the disabled branch...</Sans>
  ) : (
    <Flex>...the enabled branch...</Flex>
  )}
</Header>
```

Reply:
Sounds good, updated